package.json

```diff
@@ -87,31 +35,117 @@
       ],
       "items": {
         "type": "string"
-      }
+      },
+      "order": 3
     },
-    "rulesToSilenceWhileTyping": {
-      "title": "Silence specific rules while typing",
-      "description": "Useful when Atom fixes errors on save like `no-trailing-spaces` or `eol-last`.",
+    "eslintRulesDirs": {
```

Comment:
Is there any reason this isn't in the advanced section?

Reply:
I suppose it could be.  One of the big things that made ESLint different from JSHint was that you could write your own rules and they could be included at execution time.  Once-upon-a-time the only way was by giving a directory of rules files.  Now of course there are plugins, but I don't have a great sense for how many users out there still use custom rules in a rules directory.

Reply:
Interesting, I guess I just assumed that anyone writing their custom rules would format them as a plugin so they would work like all other custom rules.